code/game/atoms.dm

```diff
@@ -610,6 +610,9 @@ var/list/blood_splatter_icons = list()
 /atom/proc/narsie_act()
 	return
 
+/atom/proc/ratvar_act()

```

Comment:
But why. This should be self-contained in a Clockwork Cult port, than appended onto a table construction overhaul....

Reply:
It'll be too much of a pain to port over something big like ClockCult all at once, Tiger gave me the 👍 to sneaking a bit in

Reply:
its fine to sneak it in as it can cover some bases who ever port its may miss